<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMainWidget</class>
 <widget class="QMainWindow" name="IMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrackComp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Reference Track</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QToolButton" name="toolRefTrack">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelRefTrack">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Recorded Tracks</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="toolPathTracks">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/document-open-folder.png</normaloff>:/icons/document-open-folder.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelPathTracks">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolReload">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resources.qrc">
             <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeTrackFiles">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>File</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Mean [m]</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Variance [m]</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>&gt; 30m [#]</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>&lt;15m [%]</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>&lt;10m [%]</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>&lt;5m [%]</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushCompare">
      <property name="text">
       <string>Compare</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
